pkg/cosign/git/gitlab/gitlab.go

```diff
@@ -70,6 +71,7 @@ func (g *Gl) PutSecret(ctx context.Context, ref string, pf cosign.PassFunc) erro
 		EnvironmentScope: gitlab.String("*"),
 	})
 	if err != nil {
+		ui.Infof(ctx, "WARNING: if you are using a self-hosted gitlab please set the \"GITLAB_HOST\" your server name.")
```

Comment:
If you want to prompt a warning, use `Warnf(` instead.

Reply:
@hectorj2f thanks for your comment! do you think that this kind of message should be a warning? I have doubts about that, basically, here we are suggesting to the user to configure that variable if the intention is to deploy the secret on a self-hosted GitLab instance.

Reply:
I agree the message fits more into `info` here, but we are trying to call the attention of the user. Otherwise they might not pay attention to this. 

Other than that, the `ui.Warnf` function is just a `fmt.Fprintf(w.Stderr, "WARNING: %s\n", msg)` :), that is why I suggested to change it.